src/main/java/seedu/address/model/person/Birthdate.java

```diff
@@ -0,0 +1,82 @@
+package seedu.address.model.person;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
+import java.time.LocalDate;
+import java.time.Period;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+/**
+ * Represents a Person's birthdate in the address book.
+ * Guarantees: immutable; is valid as declared in {@link #isValidBirthdate(String)}
+ */
+public class Birthdate {
+
+    public static final DateTimeFormatter DATE_FORMAT = DateTimeFormatter.ofPattern("dd-MM-yyyy");
+    public static final DateTimeFormatter DISPLAYED_DATE_FORMAT = DateTimeFormatter.ofPattern("d MMM yyyy");
```

Comment:
Yes it does account for double-digit day-of-births, it just leaves out the leading 0 when the day-of-birth is single-digit, e.g. `03-10-2010` is shown as `3 Oct 2010` instead of `03 Oct 2010`.